cmake/nrf5_bsp.cmake

```diff
@@ -44,6 +44,24 @@ list(APPEND NRF5_LIBRARY_NRF5_BOARDS_DEPENDENCIES
   nrf5_soc
 )
 
+# Board Support Package (definitions only)
+add_library(nrf5_bsp_defs INTERFACE)
```

Comment:
I was thinking about `nrf5_bsp_fwd` as it's commonly used, but I guess it's fine.